src/sys/unix/selector/epoll.rs

```diff
@@ -41,7 +41,7 @@ impl Selector {
     }
 
     pub fn try_clone(&self) -> io::Result<Selector> {
-        syscall!(dup(self.ep)).map(|ep| Selector {
+        syscall!(fcntl(self.ep, libc::F_DUPFD_CLOEXEC)).map(|ep| Selector {
```

Comment:
I should have caught this reviewing this previously but, @Tim-Zhang you need to pass a third argument to `fcntl`here. It's fixed in https://github.com/tokio-rs/mio/commit/2883f5c1f35bf1a59682c5ffc4afe6b97d7d6e68 and https://github.com/tokio-rs/mio/pull/1498.

Reply:
sorry for the mistake, I saw you have fixed this, thanks